packages/react-native/ReactAndroid/src/main/java/com/facebook/react/uimanager/TransformHelper.java

```diff
@@ -60,6 +60,10 @@ public static void processTransform(ReadableArray transforms, double[] result) {
       return;
     }
 
+    float[] offsets = getTranslateForTransformOrigin(viewWidth, viewHeight, transformOrigin);
+    MatrixMathHelper.applyTranslate3D(helperMatrix, offsets[0], offsets[1], offsets[2]);
+    MatrixMathHelper.multiplyInto(result, result, helperMatrix);
```

Comment:
`result` is the identity matrix at this point, no multiplication should be required to do this.